src/components/GlobalFilter/GroupFilterInputGroup.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FormEvent, MouseEventHandler } from 'react';
+import { useIntl } from 'react-intl';
+import messages from '../../locales/Messages';
+import { FilterMenuItemOnChange } from '@redhat-cloud-services/frontend-components/ConditionalFilter/groupFilterConstants';
+import { Group } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
+import { GroupFilter } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
+import { Button, InputGroup, InputGroupItem, Popover, PopoverPosition } from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { FlagTagsFilter } from '../../@types/types';
+
+/** Create unique hotjar event for selected tags */
+const generateGlobalFilterEvent = (isChecked: boolean, value?: string) => `global_filter_tag_${isChecked ? 'uncheck' : 'check'}_${value}`;
+
+export type GroupFilterInputGroupProps = {
+  isDisabled: boolean;
+  filter: {
+    filterBy?: string | number;
+    onFilter?: (value: string) => void;
+    groups?: Group[];
+    onChange: FilterMenuItemOnChange;
+  };
+  setIsOpen: (callback?: ((origValue?: boolean) => void) | boolean) => void;
+  hotjarEventEmitter?: ((eventType: string, eventName: string) => void) | (() => void);
+  selectedTags: FlagTagsFilter;
+};
+
+const GroupFilterInputGroup: React.FunctionComponent<GroupFilterInputGroupProps> = ({ filter, isDisabled, selectedTags, setIsOpen }) => {
+  const intl = useIntl();
+
+  return (
+    <InputGroup>
+      <InputGroupItem>
+        <GroupFilter
+          className="chr-c-menu-global-filter__select"
+          selected={selectedTags}
+          isDisabled={isDisabled}
+          groups={
+            filter.groups?.map((group) => ({
+              ...group,
+              items: group.items?.map((item) => ({
+                ...item,
+                onClick: (
+                  e: FormEvent | MouseEventHandler<HTMLInputElement> | undefined,
+                  selected: any,
+                  group: number | undefined,
+                  currItem: boolean | undefined,
+                  groupName: string | undefined,
+                  itemName: string
+                ) => {
+                  generateGlobalFilterEvent((selected?.[groupName as string]?.[itemName as string] as Group)?.isSelected as boolean, item.value);
+                  // item.onClick?.(e, selected, group, currItem, groupName, itemName);
```

Comment:
Do we want to disable the `onClick` event?
```suggestion
                  item.onClick?.(e, selected, group, currItem, groupName, itemName);
```

Reply:
This was all due to a hotjar and besides the condition was always false. Deleted all the unnecessary code.